# Probability Theory and Mathematical Statistics

## Homework 1, Vitaliy Pozdnyakov

___

### Part 1

Prove mathematically via definitions and axioms

**1. Let P(A) = P(B) = ½. Is it true that P(A|B) = P(B|A)?**

$P(A|B) = \frac{P(AB)}{P(B)} = 2 \cdot P(AB)$

$P(B|A) = \frac{P(AB)}{P(A)} = 2 \cdot P(AB)$

$P(A|B) = P(B|A)$

Answer: It is true

**2. Let $H_1$, $H_2$, $H_3$, $H_4$ be equally probable hypotheses. Are the event $H_1 + H_2$ and the event $H_3 + H_4$ also hypotheses?**

$H_1, H_2, H_3, H_4$ do not intersect and do union to $\Omega$ via definition:

$H_1 + H_2 + H_3 + H_4 = \Omega$

Let $H_A = H_1 + H_2$ and $H_B = H_3 + H_4$

If $H_1, H_2, H_3, H_4$ do not intersect, then $H_A$ and $H_B$ do not intersect too.

$H_A + H_B = \Omega$ 

Thereby we got a system of hypotheses

Answer: Yes

**3. Let P(B) > 0. Prove that P(¬A|B) = 1 - P(A|B)**

$P(A|B) + P(\bar{A}|B) = \frac{P(AB)}{P(B)} + \frac{P(\bar{A} B)}{P(B)} = \frac{P(A B) + P(\bar{A} B)}{P(B)} = \frac{P((A B) + (\bar{A} B))}{P(B)} = \frac{P((A + \bar{A}) B)}{P(B)} = \frac{P(\Omega B)}{P(B)} = \frac{P(B)}{P(B)} = [P(B)>0] = 1$

$P(\bar{A}|B) = 1 - P(A|B)$

**4. Let P(C) = ½. Compare the probabilities: P(A+B|C) and P(A|C) and P(B|C)**

$P(A|C) = \frac{P(AC)}{P(C)} = 2 \cdot P(AC)$

$P(B|C) = \frac{P(BC)}{P(C)} = 2 \cdot P(BC)$

$P(A+B|C) = \frac{P((A+B)C)}{P(C)} = 2 \cdot P((A+B)C) = 2P(AC+BC) = 2 (P(AC) + P(BC) - P(ABC)) = 2P(AC) + 2P(BC) - 2P(ABC) = P(A|C) + P(B|C) - 2P(ABC)$

$P(ABC) \leq P(AC)$ and $P(ABC) \leq P(BC)$, then 

$2P(ABC) \leq P(A|C)$ and $2P(ABC) \leq P(B|C)$

And thereby 

$P(A+B|C) \geq P(A|C)$

$P(A+B|C) \geq P(B|C)$

$P(A|C)$ and $P(B|C)$ are not comparable

**5. Let $H_1, H_2, H_3$ be equally probable hypotheses. The event $A = H_1 + H_2$ has been happened. Are the $H_1 + H_2$ and $H_3$ the system of hypotheses?**

$H_1, H_2, H_3$ do not intersect and do union to $\Omega$ via definition:

$H_1 + H_2 + H_3 = \Omega$

$A = H_1 + H_2$

If $H_1, H_2, H_3$ do not intersect, then $A$ and $H_3$ do not intersect too.

$A + H_3 = \Omega$

Thereby we got a system of hypotheses

Answer: Yes

___

### Part 2

**1. One card has been randomly picked up from the deck of cards (52 cards). Are the events A and B dependent? A = {the card is ace}, B = {the card is peak suit}**

$P(A) = \frac{4}{52}$

$P(B) = \frac{1}{4}$

$P(A|B) = \frac{P(AB)}{P(B)} = \frac{\frac{1}{52}}{\frac{1}{4}} = \frac{4}{52} = P(A)$

$P(B|A) = \frac{P(AB)}{P(A)} = \frac{\frac{1}{52}}{\frac{4}{52}} = \frac{1}{4} = P(B)$

Answer: $A$ and $B$ are independent

**2. Someone has been found the credit card. What is the probability that the pin-code (4 digits) can bee guessed with two attempts?**

Let $A_i$ — successful i-th attempt

Find $P(\bar{A_1}A_2)$

$A_1$ and $A_2$ are dependent because after the first failed attempt the number of undefined combinations will change.

$\tilde{A}^4_{10} = 10^4$ - the number of all available combinations, then

$P(\bar{A_1}A_2) =  P(\bar{A_1}) \cdot P(A_2|\bar{A_1})= \frac{10^4 - 1}{10^4} \cdot \frac{1}{10^4-1} = \frac{1}{10^4}$

Answer: $\frac{1}{10^4}$

**3. The average percent of loan default rate is 5%. What is the probability that the problems with loan repayment occur at least 2 times considering that the bank has issued 100 loans?**

Let $A_i$ — problems occur i times

Find $P(A_2 + A_3 + \dots + A_{100})$

$P(A_2 + A_3 + \dots + A_{100}) = 1 - P(A_0 + A_1)$

$P(A_n) = С^n_{100} \cdot 0.05^n \cdot 0.95^{100-n}$ — binomial distribution

$P(A_0 + A_1) = P(A_0) + P(A_1) = C^0_{100}0.05^00.95^{100} + C^1_{100}0.05^10.95^{99} = 0.95^{100} + 5 \cdot 0.95^{99}$

$1 - P(A_0 + A_1) = 1 - (0.95^{100} + 5 \cdot 0.95^{99})$ 

In [2]:
1 - (0.95**100 + 5 * (0.95**99))

0.962918790672645

Answer: $\approx 0.96$

**4. The retail company has laptops of three manufactures with proportion of 2:5:3. The laptops by Manufacture 1 requires the repair during the warranty period in 15% cases; The laptops by Manufacture 2 requires the repair during the warranty period in 8% cases; The laptops by Manufacture 3 requires the repair during the warranty period in 6% cases. What is the probability that a new laptop received by retail company will require the repair during the warranty period?**

Let $A_i$ — received laptop is from Manufacture i, and $B$ – recieved laptop require the repair

Find $P(B)$

We have

$P(A_1) = \frac{2}{2+5+3} = \frac{2}{10}$; $P(A_2) = \frac{5}{10}$; $P(A_2) = \frac{3}{10}$

$P(B|A_1) = 0.15$; $P(B|A_2) = 0.08$; $P(B|A_1) = 0.06$

Then

$P(B) = P(A_1B) + P(A_2B) + P(A_3B) = P(A_1)P(B|A_1) + P(A_2)P(B|A_2) + P(A_2)P(B|A_2) = \frac{2}{10}0.15 + \frac{5}{10}0.08 + \frac{3}{10}0.06 = \frac{30}{1000} + \frac{40}{1000} + \frac{18}{1000} = \frac{88}{1000}$ 

Answer: 0.088

**5. After exanimation of a patient the doctor considers diseases C and D with equal probability. To clarify the diagnosis the doctor assign one more test. If the disease is C, so the result of the test is positive in 30% of cases. If the disease is D, so the result of the test is positive in 20% of cases. What disease is more likely?**

Let $I$ — the test indicates positive result,

$C$ and $D$ — the patient's desease is C and D respectively

Suppose that $I$ happended, so find and compare $P(C|I)$ and $P(D|I)$

We have

$P(C) = P(D) = 0.5$

$P(I|C) = 0.3$; $P(I|D) = 0.2$

Via Bayes' theorem:

$P(C|I) = \frac{P(I|C)P(C)}{P(I)} = \frac{0.5 \cdot 0.3}{P(I)} = \frac{0.15}{P(I)}$ and

$P(D|I) = \frac{P(I|D)P(D)}{P(I)} = \frac{0.5 \cdot 0.2}{P(I)} = \frac{0.1}{P(I)}$ 

$\frac{0.15}{P(I)} > \frac{0.1}{P(I)} \Rightarrow P(C|I) > P(D|I)$

Answer: Desease $C$ is more likely than desease $D$